lib/FusionInventory/Agent/Task.pm

```diff
@@ -59,6 +59,14 @@ sub getModules {
     return @modules
 }
 
+sub isRemote {
```

Comment:
This is a mixed get/set attribute handler, whereas we had distinct handlers sofar. Moreover, the name is quite unfortunate for a setter method. I'd rather recommend using isRemote() a getter, setRemote() as setter.

Reply:
Okay, I'll update the PR accordingly.